templates/src-chain-templates/__network__/decoders/calls/identities/setIdentity.ts

```diff
@@ -0,0 +1,22 @@
+// import { calls } from '../../../types';
+// import { UnknownVersionError } from '../../../../../utils';
+// import { Call, ISetIdentityCallPalletDecoder } from '../../../../../indexer';
+
+// export class SetIdentityCallPalletDecoder implements ISetIdentityCallPalletDecoder {
+//   decode(call: Call) {
+//     let identity = calls.identity.setIdentity;
+
+//     if (identity.v1030.is(call)) {
+//       return {
+//         twitter: {
+//           __kind: 'None',
+//         },
+//         ...identity.v1030.decode(call).info,
+//       };
+//     } else if (identity.v1032.is(call)) {
+//       return identity.v1032.decode(call).info;
+//     } else {
+//       throw new UnknownVersionError(identity);
+//     }
+//   }
+// }
```

Comment:
4 files in the templates dir are entirely commented out.

Reply:
Yeah, they are not valid `ts` files since they are outside of `src` folder and the main purpose is to show an example of pallet implementation. I'll uncomment them and make typescript checker to skip these files